suite-native/module-accounts-import/src/components/XpubScanHeader.tsx

```diff
@@ -1,63 +1,44 @@
 import React from 'react';
-import { View } from 'react-native';
 
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box, Chip, Text } from '@suite-native/atoms';
-import { CryptoIcon } from '@trezor/icons';
+import { Box, Select, Text } from '@suite-native/atoms';
 import { networks, NetworkSymbol } from '@suite-common/wallet-config';
-import { enabledNetworks } from '@suite-native/config';
+import { CryptoIconName } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
 type XpubScanHeaderProps = {
     onSelectCurrency: (currencySymbol: NetworkSymbol) => void;
     selectedCurrencySymbol: NetworkSymbol;
 };
 
-const chipsWrapperStyle = prepareNativeStyle(utils => ({
-    flexDirection: 'row',
-    borderRadius: utils.borders.radii.large,
-    backgroundColor: utils.colors.gray200,
-}));
-
-const chipStyle = prepareNativeStyle<{ isSelected: boolean }>((utils, { isSelected }) => ({
-    ...utils.typography.label,
-    flex: 1,
-    backgroundColor: 'transparent',
-    borderColor: 'transparent',
-    borderRadius: utils.borders.radii.large,
-    extend: {
-        condition: isSelected,
-        style: {
-            backgroundColor: utils.colors.gray0,
-        },
-    },
+const selectWrapper = prepareNativeStyle(() => ({
+    width: '100%',
 }));
 
 export const XpubScanHeader = ({
     onSelectCurrency,
     selectedCurrencySymbol,
 }: XpubScanHeaderProps) => {
     const { applyStyle } = useNativeStyles();
+
+    const selectNetworkItems = Object.keys(networks).map(networkSymbol => ({
```

Comment:
We should use our enabled networks config to filter these options because there is great possibility that some of the coins will not work correctly in app and we will not wan't to show them etc.

Reply:
https://github.com/trezor/trezor-suite/pull/6970/commits/173672e9f21838f6518e120d7bb490e306a1b8bd